lms/djangoapps/completion/tests/test_models.py

```diff
@@ -142,3 +142,48 @@ def test_cannot_call_submit_completion(self):
                 completion=0.9,
             )
         self.assertEqual(models.BlockCompletion.objects.count(), 1)
+
+
+class SubmitBatchCompletionTestCase(TestCase):
+    """
+    Test that BlockCompletion.objects.submit_batch_completion has the desired
+    semantics.
+    """
+
+    def setUp(self):
+        super(SubmitBatchCompletionTestCase, self).setUp()
+        _overrider = waffle.waffle().override(waffle.ENABLE_COMPLETION_TRACKING, True)
+        _overrider.__enter__()
+        self.addCleanup(_overrider.__exit__, None, None, None)
+
+        self.block_key = UsageKey.from_string('block-v1:edx+test+run+type@video+block@doggos')
+        self.course_key_obj = CourseKey.from_string('course-v1:edx+test+run')
+        self.user = UserFactory()
+        CourseEnrollmentFactory.create(user=self.user, course_id=unicode(self.course_key_obj))
+
+    def test_submit_batch_completion(self):
+        blocks = [(self.block_key, 1.0), ]
```

Comment:
You don't need to use trailing commas unless the list is defined across multiple lines.